schema/site.schema.json

```
@@ -381,6 +381,19 @@
           "description": "Enable filtering of all exec commands on gitserver based on a pre-defined allowlist",
           "type": "boolean",
           "default": true
+        },
+        "gitServerPinnedRepos": {
+          "description": "List of repositories pinned to specific gitserver instances. The specified repositories don't get re-cloned elsewhere on scaling the gitserver cluster.",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "examples": [
+            {
+              "repo1": "gitserver1",
```

Because this is the only documentation we have for now, we might want to be a little bit more explicit. How about something like this?

```suggestion
              "github.com/foo/bar": "gitserverHostname",
```

---

Fixed in [da6783a](https://github.com/sourcegraph/sourcegraph/pull/32831/commits/da6783a43c50d03293a4633884d1ba9f94d08daf)